modules/gff_reduce.nf

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+process GFF_REDUCE {
+    publishDir "$launchDir/$params.outdir/gff_output_files"
```

Comment:
The launchDir should not be part of the outdir, for example if the user specified a full path for the outdir it will not work

Reply:
Agree. I have removed $launchDir in all the module scripts